### PR TITLE
Change Lichess To Lishogi

### PR DESCRIPTION
**Pull request by lidevelopers**

### CODE REVIEW

File: lishogi [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-# Starts a dev console to compile and run lichess.
+# Starts a dev console to compile and run lishogi.
 
 # Usage:
 # ./lila
```
Comment (TheYoBots) [inline lishogi:6]: `./lishogi` here instead of `./lila`

**Comment by WandererXII** [comment]
Probably later, since this is more consistent, but I have a lot of scripts that use the name 'lila' so I also need to change all that.

**Comment by lidevelopers** [comment]
@WandererXII I have change some more scripts lila to lishogi can you check if that fine

**Comment by lidevelopers** [comment]
And I don't know if I edit these files will there be errors still I didn't update changes in my local env so if there any errors please if you can edit my commits

**Pull request closed without merging**